SECP-service/src/main/resources/assets/app/views/portal/configure-tags.html

```diff
@@ -0,0 +1,28 @@
+<body>
+<h1 class="portalheader"><i class="fa fa-tags" aria-hidden="true"></i> Tags</h1>
+<p class="note"><strong>Note:</strong> Add and delete permission levels and user roles by clicking below.</p>
+<div>
+    <br>
+    <p class="tag-option"><a class="configureoption" ng-click="permissionClick()"><span class="optionicon" id="optionicon1">+</span><i class="fa fa-low-vision" aria-hidden="true"></i> Permission Levels</a></p>
+    <hr id="optionline1">
+    <br>
+    <p class="tag-option"><a class="configureoption" ng-click="rolesClick()"><span class="optionicon" id="optionicon2">+</span><i class="fa fa-id-card-o" aria-hidden="true"></i> Roles</a></p>
+    <hr id="optionline2">
+</div>
+<div id="permissionTableDiv" style="display: none;">
```

Comment:
Would be cool if you put the display: none in css file but doesnt matter

Reply:
Agreed ✔️